Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -140,34 +156,28 @@ export RGW_POOL_PREFIX=default
     rook-ceph-external   /var/lib/rook                162m   Connected   HEALTH_OK
     ```
 
-6. StorageClass will also be created, verify its creation. `ceph-rbd` and `cephfs` StorageClass would be respective name for RBD and CephFS StorageClass.
+6.  Verify the creation of the storage class depending on the rbd pools and filesystem provided.
+    `ceph-rbd` and `cephfs` would be the respective names for the RBD and CephFS storage classes.
     ```console
     kubectl -n rook-ceph-external get sc
     ```
 
 7. Then you can now create a [persistent volume](https://github.com/rook/rook/tree/master/deploy/examples/csi) based on these StorageClass.
 
-### CephCluster example (management)
-
-The following CephCluster CR represents a cluster that will perform management tasks on the external cluster.
-It will not only act as a consumer but will also allow the deployment of other CRDs such as CephFilesystem or CephObjectStore.
-You would need to inject the admin keyring for that.
-
-The corresponding YAML example:
-
-```yaml
-apiVersion: ceph.rook.io/v1
-kind: CephCluster
-metadata:
-  name: rook-ceph-external
-  namespace: rook-ceph-external
-spec:
-  external:
-    enable: true
-  dataDirHostPath: /var/lib/rook
-  cephVersion:
-    image: quay.io/ceph/ceph:v17.2.6 # Should match external cluster version
-```
+#### Connect to an External Object Store
```

Comment:
```suggestion
### Connect to an External Object Store
```

Reply:
Will discuss this levels,
I think we should maintain the subdirectories level

Reply:
Ok let's discuss. Titles that go too deep just usually aren't necessary IMO.

Reply:
I still think level 3 header makes sense